app/serializers/event_date_serializer.rb

```diff
@@ -30,4 +30,11 @@ class EventDateSerializer < ActiveModel::Serializer
   attribute :date do
     Date.parse(object.date.to_s)
   end
+
+  attribute :event do
+    {
+      name: object.event.agency.loc_nickname,
```

Comment:
We want to use the loc_name field. The loc_nickname is for internal system use (PantryTrak and within MOF) e.g. "NSI" , but the loc_name is for external use (FreshTrak!) e.g. "Neighborhood Services, Inc".